frontend/react/src/pages/Auth/presentation/SignUp.jsx

```diff
@@ -73,11 +73,20 @@ function SignUp() {
     }
   };
 
+  // Navigate to home page when logo icon is clicked
+  const handleLogoClick = () => {
+    navigate("/");
+  };
+
   return (
     <>
       <section className="float-right relative h-screen w-screen lg:w-[40%]">
-        {/* Top Left Icon and Text */}
-        <div className="absolute right-3 top-3">
+        {/* Top Right Icon and Text */}
+        <div 
+          className="absolute right-3 top-3" 
+          onClick={handleLogoClick} 
```

Comment:
No need to create a seperate function just for a navigate, feel free to use anonymous function in both login and signup pages.